docs/src/pages/guides/composition/composition.md

```diff
@@ -132,7 +132,27 @@ You can find the details in the [TypeScript guide](/guides/typescript#usage-of-c
 Some components such as `ButtonBase` (and therefore `Button`) require access to the 
 underlying DOM node. This was previously done with `ReactDOM.findDOMNode(this)`.
 However `findDOMNode` was deprecated (which disqualifies its usage in react's concurrent mode)
-in favour of component refs and ref forwarding. If you pass class components to
+in favour of component refs and ref forwarding. 
+
+It is therefore necessary that the components you pass to the `component` prop
+can hold refs. This includes:
+- class components
```

Comment:
The markdown specification is fuzzy about this. Crowdin requires a new blank line when starting a list.

```suggestion

- class components
```